nodejs/eks/nodegroup.ts

```diff
@@ -565,16 +572,16 @@ async function computeWorkerSubnets(parent: pulumi.Resource, subnetIds: string[]
 /**
  * Iterates through the tags map creating AWS ASG-style tags
  */
-function tagsToAsgTags(tags: {[key: string]: string}): pulumi.Output<string> {
-    let output: pulumi.Input<string> = "";
-    for (const tag of Object.keys(tags)) {
-        output = pulumi.concat(output, pulumi.all([
-            tag,
-            tags[tag],
-        ]).apply(([k, v]) => `
-                          - Key: ${k}
-                            Value: ${v}
-                            PropagateAtLaunch: 'true'`));
-    }
-    return pulumi.output(output);
+function tagsToAsgTags(tagsInput: pulumi.Input<InputTags>): pulumi.Output<string> {
+    const out = pulumi.output(tagsInput);
```

Comment:
Is `out` used anywhere?

Reply:
Oops good catch - I was doing some experimenting and forgot to rm this line. thx! 🎉